sprite.go

```diff
@@ -251,15 +251,15 @@ func (p *SpriteImpl) init(
 	// bind physic config
 	p.collisionMask = parseLayerMaskValue(spriteCfg.CollisionMask)
 	p.collisionLayer = parseLayerMaskValue(spriteCfg.CollisionLayer)
-	p.triggerMask = parseLayerMaskValue(spriteCfg.TriggerMask)
-	p.triggerLayer = parseLayerMaskValue(spriteCfg.TriggerLayer)
-
-	p.colliderType = paserColliderType(spriteCfg.ColliderType)
+	// collider is disable by default
```

Comment:
Does this mean we need to specify something like `colliderType: "auto"` to explicitly enable collision-features like touch-check?

Reply:
No , to keep consistency with spx1, the trigger is enabled by default, and collision is disabled by default. you don't need to config colliderType